client/v2/client_test.go

```diff
@@ -257,16 +258,19 @@ func TestClient_Concurrent_Use(t *testing.T) {
 	wg.Add(3)
 	n := 1000
 
+	errC := make(chan error, 3)
```

Comment:
The problem with your approach is that `wg.Wait` will never get done because the second goroutine could be stuck trying to send on an unbuffered channel.